core/src/main/java/com/cerner/beadledom/core/BeadledomModule.java

```diff
@@ -37,14 +33,10 @@
 public class BeadledomModule extends AbstractModule {
   @Override
   protected void configure() {
-    install(new AvroJacksonGuiceModule());
-    install(new AvroSwaggerGuiceModule());
     install(new BeadledomConfigurationModule());
     install(new HealthModule());
```

Comment:
Should the health module be opt-in as well?

Reply:
I think it should, @johnlcox ?

Reply:
Made the HealthModule also opt-in, https://github.com/cerner/beadledom/pull/54/commits/866182169cdd0c5237b5ca77245fe9fe3f20b0ee